src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx

```diff
@@ -27,23 +27,53 @@ import { Title } from 'lib/ui/typography/Title.js'
 
 export interface AccessCodeTableProps {
   deviceId: string
+  accessCodeFilter?: (
+    accessCode: AccessCode,
+    searchInputValue: string
+  ) => boolean
+  accessCodeComparator?: (
+    accessCodeA: AccessCode,
+    accessCodeB: AccessCode
+  ) => number
   onBack?: () => void
   className?: string
 }
 
+type AccessCode = UseAccessCodesData[number]
+
+const defaultAccessCodeFilter = (
+  accessCode: AccessCode,
+  searchInputValue: string
+) => {
+  if (searchInputValue === '') return true
```

Comment:
🐠 Pretty minor, but we could add a `.trim()` here unless there's a reason someone would search for an empty space(s)